frontend/src/app/app.module.ts

```diff
@@ -22,7 +23,9 @@ import { ToastrModule } from 'ngx-toastr';
     WorkSpaceModule,
     ShellModule,
     BrowserAnimationsModule,
-    ToastrModule.forRoot()],
+    ToastrModule.forRoot(),
+    UserModule
```

Comment:
Remove UserModule since it's going to be lazy-loaded through the routing. We don't need  direct dependencies